sql-plugin/src/main/311until330-all/scala/com/nvidia/spark/rapids/shims/GpuTypeShims.scala

```diff
@@ -80,4 +80,19 @@ object GpuTypeShims {
 
   def csvRead(cv: cudf.ColumnVector, dt: DataType): cudf.ColumnVector =
     throw new RuntimeException(s"Not support type $dt.")
+
+  /**
+   * Spark supports interval type from 320, but GPU from 330, so just return false
+   */
+  def isDayTimeInterval(dt: DataType) : Boolean = false
+
+  /**
+   * Spark supports interval type from 320, but GPU from 330, so just return false
+   */
+  def isYearMonthInterval(dt: DataType) : Boolean = false
+
+  /**
+   * Spark supports interval type from 320, but GPU from 330, so just return false
+   */
+  def isAnsiInterval(dt: DataType) : Boolean = false
```

Comment:
Personally this method is somewhat duplicate of isYearMonthInterval and isDayTimeInterval.
It is equal to `isDayTimeInterval || isYearMonthInterval`. So I would prefer to removing it.
Yes I can reduce the code, but cause more confusion. It implies there would other ansi intervals besides year month and day time.